nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -735,8 +735,13 @@ private static Type getTypeForSymbol(Symbol symbol, VisitorState state) {
     }
   }
 
-  private static Nullness getGenericMethodReturnTypeNullness(
-      Symbol.MethodSymbol method, Type enclosingType, VisitorState state, Config config) {
+  static Nullness getGenericMethodReturnTypeNullness(
+      Symbol.MethodSymbol method, @Nullable Type enclosingType, VisitorState state, Config config) {
+    if (enclosingType == null) {
+      // we have no additional information from generics, so return NONNULL (presence of a @Nullable
+      // annotation should have been handled by the caller)
+      return Nullness.NONNULL;
```

Comment:
I think you'll fix the code coverage issue here if you remove the similar code at lines 702-706 (which is no longer needed)

Reply:
Makes a lot of sense! Thanks!